module-2/cdk/lib/CoreConstruct.ts

```diff
@@ -0,0 +1,33 @@
+import ec2 = require("@aws-cdk/aws-ec2");
+import iam = require("@aws-cdk/aws-iam");
+import cdk = require("@aws-cdk/cdk");
+
+export class CoreConstruct extends cdk.Construct {
+  public readonly vpc: ec2.Vpc;
+
+  constructor(scope: cdk.Construct, id: string) {
+    super(scope, id);
+
+    this.vpc = new ec2.Vpc(this, "VPC", {
+      cidr: "10.0.0.0/16",
+      enableDnsHostnames: true,
+      enableDnsSupport: true,
+      natGateways: 1,
+      maxAZs: 2
+    });
+    const dynamoDbEndpoint = this.vpc.addGatewayEndpoint("DynamoDbEndpoint", {
+      service: ec2.GatewayVpcEndpointAwsService.DynamoDb,
+      subnets: [
```

Comment:
This is the [default](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-ec2.GatewayVpcEndpointOptions.html#subnets). You can omit